website/content/docs/commands/job/action.mdx

```diff
@@ -0,0 +1,109 @@
+---
+layout: docs
+page_title: 'Commands: job action'
+description: |
+  The job action command is used to execute predefined actions from a job
+  specification in a running task context
+---
+
+# Command: job action
+
+**Alias: `nomad action`**
+
+The `job action` command allows operators to execute predefined actions declared
+in Nomad job specifications. These actions can be defined at task level and are
+intended for specific operational tasks, such as clearing a cache, or migrating
+a database.
+
+An action may self-terminate upon completion (for example, echoing a string),
+or run for an indeterminate amount of time (for example, watching a blocking
+query). In the latter case, an action can be terminated via escape character
+(such as cmd+c / ctrl+c)
+
+When ACLs are enabled, this command requires a token with the `alloc-exec`,
+`read-job`, and `list-jobs` capabilities for the allocation's namespace. If
+the task driver does not have file system isolation (as with `raw_exec`),
+this command requires the `alloc-node-exec`, `alloc-exec`, `read-job`, and
+`list-jobs` capabilities for the allocation's namespace.
+
+## Usage
+
+```plaintext
+nomad job action [options] <job_action_name>
+```
+
+The `job action` command requires an action name and accepts two ways of
+specifying where it should be run:
+- by passing the job name and known allocation ID along with the name of the action
+- by passing the job, task group, and task name along with the name of the action
+(when the allocation ID is not known). A random allocation will be selected if
+multiple are available.
+
+The action name provided must be defined within a task in the [job specification]
+provided. With sufficient privileges, an execution context will be opened and
+the defined action command will be run. No further input is possible, save for
+the escape character to terminate execution, so interactive commands are not
+supported.
+
+## General Options
+
+@include 'general_options.mdx'
+
+## Action Options
+
+- `-job`: (Required) Specifies the job containing the predefined action. This
+  is the job under which the task is registered.
+
+- `-alloc`: Specifies the allocation within which the action is to be executed.
+  If omitted, either a task-group and task-name must be provided, and a random
+  allocation will be selected from them.
+
+- `-task`: Specifies the task within the job where the action is defined. This
+  must correspond to a task in the job's specification, and is required if no
+  `alloc` option is provided.
```

Comment:
```suggestion
- `-task`: Specifies the task within the job where the action is defined. 
```

The second sentence seems a bit redundant and I think the last one is not true? Don't you always need to specify a task, otherwise we can't find the action?

Reply:
This feedback threw me into a panic, but you're right! What I wrote is only true in instances of one-task-per-alloc, where it'll work without `-task` being passed in.

Otherwise we'll get something like this:

```
Allocation "abcd1234" is running the following tasks:
  * task
  * othertask

Please specify the task.
```

I'll modify the doc accordingly. Phew, thanks @lgfa29 !